src/ws/cockpitwebservice.c

```diff
@@ -457,11 +457,6 @@ process_socket_authorize (CockpitWebService *self,
   cockpit_memory_clear (data, length);
 
 out:
-  if (password)
-    {
-      cockpit_memory_clear (password, -1);
-      free (password);
-    }
```

Comment:
Replacing the other parts with `goto out` doesn't make much sense, since a lot of things do happen after the clearing.
@martinpitt This part is unused because in every path that has `password != NULL`, `password` is cleared before reaching this section.